src/test/java/com/powsybl/openloadflow/ac/SecondaryVoltageControlTest.java

```diff
@@ -191,6 +191,10 @@ void testUnblockGeneratorFromLimit() throws IOException {
         g6.setTargetV(11.8);
         g8.setTargetV(19.5);
 
+        // This scenario works if the slack distribution fails and an injection is added at the slack bus
+        parametersExt.setPlausibleActivePowerLimit(5000); // Remove large groups from slack distribution
```

Comment:
"group" is understandable only to french people, use "generator" instead

```suggestion
        parametersExt.setPlausibleActivePowerLimit(5000); // Remove large generators from slack distribution
```